test/io-tests/test_io.py

```diff
@@ -73,15 +73,15 @@ def defaultenv():
     }
 
 
-def dumpconfig(configpath=None, env=None, stdin=None):
+def dumpconfig(configpath=None, env={}, stdin=None):
```

Comment:
This is not quite the same, I suggest that we revert. It's good practice to use None for default arguments, as the dict in this new function definition will only be created once. If it's modified anywhere later by mistake, it would impact all other calls of the function.

Reply:
That's a left-over from some hacks that I had to use at some point inbetween. Figured I'd leave it in... I'm glad I did, learned about that problem now. Will revert it.